samples/net/openthread/coprocessor/boards/cc1352p7_lp.overlay

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Optional: You may consider moving this configuration except for the "status = okay" to the board configuration of all supported cc13xx_cc16xx boards (with proper default PIN assignemnts), so that they all support the sample (and applications derived from it) mostly ootb and then only keep the "status=okay" here. @vaishnavachath Do you see any problems with this?

Reply:
I moved everything related to uart1 to boards dts. Still, I am keeping the chosen node because I swap uart1 and uart0, which in my opinion make the application more user friendly.

Reply:
Ok, sounds good. I'll re-check once this is ready for final review.